.codeclimate.yml

```
@@ -30,6 +30,10 @@ checks:
   identical-code:
     config:
       threshold: # language-specific defaults. an override will affect all languages.
+exclude_patterns:
+  - "**/*.{gif,png,jpeg}"
```

```suggestion
  - "**/*.{gif,png,jpeg,jpg}"
```
Just in case, I think 🤔 
Do you think it makes sense?

---

Yes